tools/Runtime Condenser/Main.cpp

```diff
@@ -13,139 +13,151 @@
 #include <fstream>
 #include <string>
 #include <sstream>
+#include <vector>
+#include <algorithm>
 
 using namespace std;
 
+struct runtime {
+	string text;
+	string proc;
+	string source;
+	string usr;
+	string src;
+	string loc;
+	unsigned int count;
+};
+struct harddel {
+	string type;
+	unsigned int count;
+};
 //Make all of these global. It's bad yes, but it's a small program so it really doesn't affect anything.
-	//Because hardcoded numbers are bad :(
-	const unsigned short maxStorage = 99; //100 - 1
-
-	//What we use to read input
-	string currentLine = "Blank";
-	string nextLine = "Blank";
-
-	//Stores lines we want to keep to print out
-	string storedRuntime[maxStorage+1];
-	string storedProc[maxStorage+1];
-	string storedSource[maxStorage+1];
-	string storedUsr[maxStorage+1];
-	string storedSrc[maxStorage+1];

```

Comment:
Thats not 1, not 2, not 3, but SIX parallel global arrays. (and one more further down)